libsplinter/Cargo.toml

```diff
@@ -129,6 +129,8 @@ experimental = [
     "service-arguments-converter",
     "service-id",
     "service-lifecycle",
+    "service-lifecycle-executor",
```

Comment:
This should probably be split into a separate commit

Reply:
split